io.home-assistant.binary-sensor.battery/dataset.tt

```diff
@@ -0,0 +1,152 @@
+dataset @io.home-assistant.binary-sensor.battery language "en" {
```

Comment:
Do you not think that other products would want to implement a battery level binary sensor ?

Reply:
If you're referring to the namespace, that was my request. The protocol and naming we're standardizing on is Home Assistant-specific, so it's unclear whether they should have a generic identifier. On top of that, having a prefix helps with scripting across all these devices - and there's a lot of them so we'll need automation just manage them.